ocrd_zip.md

```diff
@@ -10,77 +10,144 @@ files such as images and metadata about those images such as PAGE or ALTO
 long-term preservation, reproducible tests etc.) it is desirable to have a
-self-contained representation of a workspace. With such a representation, data
-producers are not forced to provide dereferencable HTTP-URL for the files they
-produce and data consumers are not forced to dereference all HTTP-URL.
+self-contained representation of a [workspace](/mets).
+
+With such a representation, data producers are not forced to provide
+dereferencable HTTP-URL for the files they produce and data consumers are not
+forced to dereference all HTTP-URL.
 
 While METS does have mechanisms for embedding XML data and even base64-encoded
 binary data, the tradeoffs in file size, parsing speed and readability are too
 great to make this a viable solution for a mass digitization scenario.
 
-Instead, OCRD-ZIP is based on the widely used ZIP format which allows
-representing file hierarchies in a standardized, compressable archive format.
-Many formats like JAR (used in software development) and BagIt (used in
-long-term preservation) use the same principles: A zip file containing a
-manifest of contained resources and the resources themselves. For OCRD-ZIP, the
-METS file is the manifest.
+Instead, we propose an exchange format ("OCRD-ZIP") based on the BagIt spec
+used for data ingestion adopted in the web archiving community.
 
-## Format
+## BagIt profile
 
-### ZIP
+As a baseline, an OCRD-ZIP must adhere to [v0.97+ of the BagIt
+specs](https://tools.ietf.org/html/draft-kunze-bagit-16), i.e.
+
+* all files in `data/`
+* a file `bagit.txt`
+* a file `bagit-info.txt`
+
+In addition, OCRD-ZIP adhere to a [BagIt
+profile](https://github.com/bagit-profiles/bagit-profiles) (see [Appendix A for
+the full definition](#appendix-a)):
+
+* `bagit-info.txt` MAY additionally contain these tags:
+  * `X-Ocrd-Mets`: Alternative path to the mets.xml file if its path IS NOT `/data/mets.xml`
+  * `X-Ocrd-Manifestation-Depth`: Whether all URL are dereferenced as files or only some
+* A file `url-sources.txt` MUST exist and contain a mapping from local file name to URL
+
+### `X-Ocrd-Mets`
+
+By default, the METS file should be at `data/mets.xml`. If this file has
+another name, it must be listed here and implementations MUST check for
+`X-Ocrd-Mets` before assuming `data/mets.xml`.
+
+### `X-Ocrd-Manifestation-Depth`
 
-An OCRD-ZIP MUST be a valid ZIP file.
+Specifiy whether the bag contains the full manifestation of the data referenced in the METS (`full`)
+or only those files that were `file://` URLs before (`partial`). Default: `partial`.
 
-### `mets.xml` in the root folder
+### `url-sources.txt`
 
-The root folder of the ZIP filetree must contain a file `mets.xml`.
+Simple text file, mapping Bag-local filenames to the URL of their original location if any.
 
-### `file://`-URLs must be relative
+Every mapping must be on a new line.
+
+Every line should have the format `URL FILENAME`, i.e. a single space character between the two.
+
```

Comment:
> We could use fetch.txt for referencing original files.

If you mean the mechanism discussed in OCR-D/core#176 (`url-sources`): This is explicitly not about fetching data but different "aliases" for the same URL to allow using different names e.g. in PAGE for a file in METS. Since we're not describing different manifestations of the same data but just aliases I don't see the point of having the redundant file size information in there.